index.d.ts

```diff
@@ -132,6 +218,13 @@ export class Window<T = unknown> extends (EventEmitter as new <T>() => TypedEmit
      */
     firstEmptyInventorySlot(hotbarFirst?: boolean): number | null;
 
+    /**
+     * Returns how much items there are ignoring what the item is
+     * @param start
+     * @param end
+     */
+    sumRange(start: number, end: number): number;
```

Comment:
can you give a bit more info on what this is counting? what are start and end?

Reply:
Returns how much items there are ignoring what the item is

basically taking start slot and end slot and counting the items from start to end slot, but i think just reading the function code would make this obvious